cesm2_albedo/calculate_sw_alb.jl

```diff
@@ -0,0 +1,98 @@
+using NCDatasets
+
+# Load the data
+rsds_file = NCDataset("rsds_Amon_CESM2_historical_r1i1p1f1_gn_185001-201412.nc")
+rsus_file = NCDataset("rsus_Amon_CESM2_historical_r1i1p1f1_gn_185001-201412.nc")
+
+# Extract the data
+rsds = rsds_file["rsds"][:, :, :]
+rsus = rsus_file["rsus"][:, :, :]
+
+# Create output file
+sw_alb_file = NCDataset("sw_albedo_Amon_CESM2_historical_r1i1p1f1_gn_185001-201412_v2_no-nans.nc", "c")
+
+# Define the dimensions
+defDim(sw_alb_file, "lat", rsus_file.dim["lat"])
+defDim(sw_alb_file, "bnds", rsus_file.dim["nbnd"])
+defDim(sw_alb_file, "lon", rsus_file.dim["lon"])
+defDim(sw_alb_file, "time", rsus_file.dim["time"])
+
+# copy file attributes
+for (attrib_name,attrib_value) in rsus_file.attrib
+    sw_alb_file.attrib[attrib_name] = attrib_value
+end
+
+# for each var in the dataset, copy the needed attributes and set the others
+sw_attrib = []
+for (attrib_name,attrib_value) in rsus_file["rsus"].attrib
+    if attrib_name != "coordinates"
+        push!(sw_attrib, attrib_name => attrib_value)
+    end
+end
+
+time_bnds_attrib = []
+no_copy_time_bnds = ["calendar", "units"]
```

Comment:
are the attributes `no_copy_xx` attributes that are in the input files but not the output? it might be helpful to add a comment like "skip the calendar and units attributes" or something like that